week1/pythonWeek01.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# 1. Write a python script to print your name and age
+
+# In[ ]:
+
+
+a = input("Name")
+
+b = int(input("Age"))
+
+print("Your name is : ", a, " Your age is : ", b)
+
+
+# 2. Create a list of your 5 favorite movies and store it in the variable
+
+# In[ ]:
+
+
+favorite_movies = ['Bourne triology','Good will hunting','James bond','mission impossible','the wire']
+
+
+# 3. Write a Python program to display the first and last colors from the following list. 
+# color_list = ["Red","Green","White" ,"Black"]
+
+# In[ ]:
+
+
+color_list = ["Red","Green","White" ,"Black"]
+ 
+print(color_list[0],color_list[-1])
+
+
+# 4. Write a Python script to add a key to a dictionary
+# Sample Dictionary : {0: 10, 1: 20}
+# Expected Result : {0: 10, 1: 20, 2: 30}
+# 
+
+# In[ ]:
+
+
+sample_dictionary = {0:10, 1:20}
+print(sample_dictionary)
+sample_dictionary.update({2:30})
+print(sample_dictionary)
+
+
+# 5. Write a Python program to calculate body mass index.
+
+# In[ ]:
+
+
+height = float(input("Enter height in meters: "))
+weight = float(input("Enter weight in kg: "))
+
+# the formula for calculating bmi
+
+bmi = weight/(height**2) 
+
+
+print("Your BMI is: {0} and you are: ".format(bmi), end='')
+
+
+# 6. Guess a number game - between 1 to 9.
+
+# In[ ]:
+
+
+
+import random
+
+a=int(input("enter a number")) 
+
+b= random.randint(1, 10)
+
+while a!= b:
+
+    b= int(input('Guess a number between 1 to 10 '))
+
+print('Well guessed!')
```

Comment:
What if I enter 5 for `a` and `b` becomes 5 by random, then I never have to guess?